src/frontend/js/components/ReferralTable/types.tsx

```diff
@@ -7,7 +7,7 @@ export enum FilterColumns {
   DUE_DATE = 'due_date',
   STATE = 'state',
   UNIT = 'unit',
-  USER = 'user',
+  USER = 'users_unit_name',
```

Comment:
I think the `USER` key could change to reflect the fact that it's now units and not users we're dealing with